libminifi/test/opencv-tests/CaptureRTSPFrameTest.cpp

```diff
@@ -16,27 +16,29 @@
  * limitations under the License.
  */
 
-#include <uuid/uuid.h>
-#include <fstream>
 #include <map>
 #include <memory>
+#include <fstream>
 #include <utility>
 #include <string>
 #include <set>
-#include "FlowController.h"
-#include "../TestBase.h"
-#include "core/Core.h"
+#include <uuid/uuid.h>
+#include <iostream>
+#include <opencv2/opencv.hpp>
```

Comment:
Why do we need the test to directly include opencv headers?

Reply:
I am thinking about testing flowfile content later. Does it make sense?

Reply:
Yes, but we can include it that time. 

I think testing the content only makes sense in case we are the ones producing the content (so we mock the streaming soruce), which is a bigger task to do, keeping one line here doesn't help too much in that. :)